templates/Containerfile.module

```diff
@@ -6,6 +6,12 @@
 {{ self::print_containerfile(c) }}
 				{%- endfor %}
 			{%- endif %}
+		{%- else if type == "files" %}
```

Comment:
It is possible to manipulate the whitespace so we don't have to do weird manual white spai

Reply:
The indents are actually not necessary. I just put them in to better see the logic structure. Would you prefer we didn't indent?